src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -573,6 +573,47 @@
         az acr task update-run -r MyRegistry --run-id runId --no-archive false
 """
 
+helps['acr task credential'] = """
+type: group
+short-summary: Credential for a task
+"""
+
+helps['acr task credential add'] = """
+type: command
+short-summary: Add a credential to the task
+examples:
+  - name: Enable the system-assigned credential to an existing task. This will replace all existing user-assigned identities.
```

Comment:
It needs to updated.

Reply:
Can we elaborate like adds a custom registry login credential to the task